model/conversion.go

```diff
@@ -112,11 +114,15 @@ func ApplyYAML(yml string, pb proto.Message) error {
 
 // FromJSONMap converts from a generic map to a proto message using canonical JSON encoding
 // JSON encoding is specified here: https://developers.google.com/protocol-buffers/docs/proto3#json
-func (ps *ProtoSchema) FromJSONMap(data map[string]interface{}) (proto.Message, error) {
-	// Marshal to json bytes
-	str, err := json.Marshal(data)
+func (ps *ProtoSchema) FromJSONMap(data interface{}) (proto.Message, error) {
+	// Marshal to YAML bytes
```

Comment:
why indeed. The new CRD serializer seems to put `map[interface{}]interface{}` fields instead of `map[string]interface{}` which breaks JSON serializer but works with YAML serializer.